src/favicon.svg

```diff
@@ -0,0 +1,6 @@
+<svg width="1080" height="1080" viewBox="0 0 1080 1080" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
isn't there a limitation and favicon must be .ico file? at least for some browsers?

Reply:
Yep. https://caniuse.com/link-icon-svg This is why there are several `<link rel="icon" ... />` instances (line 32 is a fallback for line 33).